### PR TITLE
pick up method order form .eslintrc if possible

### LINKED ISSUE #44: sort-comp customization
It would be nice to pick up the specific sort order defined in .eslintrc if not provided through the CLI options before falling back to the airbnb order.

### PR DESCRIPTION
**Pull request by reergymerej**
fixes #44

### CODE REVIEW

File: transforms/sort-comp.js [hunk 1]
```diff
@@ -173,3 +173,22 @@ function getCorrectIndex(methodsOrder, method) {
...
 }
+
+function getMethodsOrderFromEslint(filePath) {
+  const CLIEngine = require('eslint').CLIEngine;
+  const cli = new CLIEngine({
+    useEslintrc: true,
+  });
+
+  const config = cli.getConfigForFile(filePath);
+  const { rules } = config;
```
Comment (cpojer) [inline transforms/sort-comp.js]: can you remove the spaces here? :)

**Comment by cpojer** [comment]
this is great!

**Comment by reergymerej** [comment]
Yeah, I'll fix it ASAP.

**Comment by reergymerej** [comment]
I tried to follow the surrounding styles since I didn't see any style guidelines anywhere.  Please let me know if this needs to be changed again.

**Comment by cpojer** [comment]
thank you :)

**Comment by turadg** [comment]
Yay! Here's an update for the README, #49

**Comment by TSMMark** [comment]
This is awesome. It would be great to print out any errors that occur during loading of the `.eslintrc`. I was spending a lot of time trying to figure out why it wasn't honoring my eslintrc and turns out it was erroring on a missing npm module, so it wasn't even loading my eslintrc at all, but I would never know it unless I edit this package source directly on my machine.

**Pull request merged**